### PR TITLE
Updates to search tile

### PR DESCRIPTION
**Pull request by lekshmimallika-aot**
Issue: /bcgov/entity#13673

### CODE REVIEW

File: components/BCRegistriesAccount.vue [hunk 1]
```diff
@@ -201,25 +201,15 @@
                 are currently available to search. Other business types will be added in the future.
```
Comment (kialj876) [inline components/BCRegistriesAccount.vue:201]: Not sure if this was in your ticket but this tooltip text is wrong I think? People can search for all types of businesses

**Review by kialj876 — APPROVED** [review]
Looks good to me. Just have the one question but feel free to merge if that's not valid

File: components/BCRegistriesAccount.vue (hunk 1 above)
Comment (lekshmimallika-aot) [reply components/BCRegistriesAccount.vue:201]: It is not mentioned in the ticket. I will ask Janis

**Review by doug-lovett — APPROVED** [review]
LGTM.

**Review by kialj876 — APPROVED** [review]

**Review by doug-lovett — APPROVED** [review]

**Pull request merged**